library/geonetwork

```diff
@@ -11,10 +11,10 @@ Tags: 3.0.5-postgres, 3.0-postgres
 GitCommit: e5cf5da76f557481c35b4a4d3e3cac77768a1302
 Directory: 3.0.5/postgres
 
-Tags: 3.2.1, 3.2, latest
-GitCommit: 507a1d8b02f15615566f7ac45662dbf3bc4d3649
-Directory: 3.2.1
+Tags: 3.2.2
```

Comment:
Did you want to keep the `3.2` and `latest` tags here and the `3.2-postgres` and `postgres` tags below?

Reply:
@yosifkit Yes, I want to keep them: please check my latest [commit](https://github.com/docker-library/official-images/pull/3455/commits/e7bd2c24ba9c4ec659f7b8e4972845867f6569bd). Thanks for pointing that out! +1